internal/cmd/egctl/collect.go

```diff
@@ -74,7 +73,7 @@ func runCollect(collectOpts collectOptions) error {
 	basename := ""
 	if collectOpts.outPath != "" {
 		// use override output path
-		overridePath, err := convert.ValidateOutputPath(collectOpts.outPath)
+		overridePath, err := tb.ValidateOutputPath(collectOpts.outPath)
```

Comment:
i suggest adding an internal function to call `filepath.Abs` + `os.Stat` to validate, this feels like a common util func within nothing proprietary about it, and we can avoid the entire reference altogether